arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -487,18 +487,27 @@
           "type": "string",
           "description": "Required. Indicates the type of cognitive service account.",
           "enum": [
+            "Academic",
+            "Bing.Autosuggest",
+            "Bing.Search",
+            "Bing.Speech",
+            "Bing.SpellCheck",
             "ComputerVision",
+            "ContentModerator",
             "Emotion",
             "Face",
             "LUIS",
             "Recommendations",
+            "SpeakerRecognition",
             "Speech",
+            "SpeechTranslation",
             "TextAnalytics",
+            "TextTranslation",
             "WebLM"
           ],
           "x-ms-enum": {
             "name": "Kind",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
This is changed intentionally. There are more and more APIs (Kind) onboarding to "Cognitive Services", we don't want to change Swagger->SDK->PS simply because a new API (Kind) / SKU (Pricing tier) is added. 
In other words, we want to allow end users to pass arbitrary string as Kind/SKU and response a proper error message when Kind/SKU or the combination do not exist.